src/utils/isBrowser.ts

```diff
@@ -1,8 +1,4 @@
-/**
- * Though the process variable can be redefined, this approach is suitable for now.
- * 
- * Another option is to use new Function() constructor so the execution scope of this is binded to the global scope.
- * However, this fails if Content-Security-Policy (CSP) script-src does not allow 'unsafe-eval'.
- */
-export const isBrowser = () =>
-  typeof process === "undefined" || process.release?.name.search(/node|io.js/) === -1;
+// We were previously checking if were inBrowser because there was no way to setup Signer in node.
+// However, we can now set the signer by passing in the privateKey and so this check isn't needed.
+// TODO: Remove references to this function. One thing to check is that NATS works in node.
+export const isBrowser = () => true;
```

Comment:
@jrhender Wouldn't it be better to exclude isBrowser check at all?

Reply:
@JGiter I left in `isBrowser` because I believe NATS will only work in browser.